### PR TITLE
feat: first implementation of service management via operators signatures

### PR DESCRIPTION
**Pull request by kupermind**
- Operators can sign `registerAgents()` and `unbond()` transactions for service owners to act on their behalf;
- Added tests.;
- Coverage is back to 100%.

### CODE REVIEW

**Review by mariapiamo — COMMENTED** [review]
to check compiler warnings

File: contracts/utils/OperatorSignedHashes.sol [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    /// @return True, if the signature verification is successful.
+    function _verifySignedHash(
+        address operator,
+        bytes32 msgHash,
+        bytes memory signature
+    ) internal view returns (bool)
+    {
+        // Check for the operator zero address and for the signature length
+        if (operator == address(0) || signature.length < 65) {
+            return false;
```
[the remark this answers is not in the document]
Comment (kupermind) [reply contracts/utils/OperatorSignedHashes.sol:78]: This is by design, this function just returns true or false signature comparison against the hash

[the remark this answers is not in the document]
Comment (kupermind) [reply contracts/utils/OperatorSignedHashes.sol:78]: But we can refactor to have reverts in all the places

[the remark this answers is not in the document]
Comment (mariapiamo) [reply contracts/utils/OperatorSignedHashes.sol:78]: I think this would be good. What do you think @77ph?

File: test/GnosisSafeMultisig.js [hunk 2]
```diff
@@ -31,15 +31,6 @@ describe("GnosisSafeMultisig", function () {
...
     context("Creating multisigs", async function () {
-        it("Passing the zero data value", async function () {
-            // Pass the default multisig owner addresses, threshold and a zero input
-            const tx = await gnosisSafeMultisig.create(defaultOwnerAddresses, threshold, "0x");
-            const result = await tx.wait();
-
-            // Check that the obtained address is not zero
-            expect(result.events[0].address).to.not.equal(AddressZero);
-        });
-
```
Comment (kupermind) [inline test/GnosisSafeMultisig.js:42]: This test was failing another test from the opeartor signature transaction... I could not explain this, but this test is redundant as well, it just creates an empty multisig.

**Review by mariapiamo — APPROVED** [review]

**Pull request merged**